types/imagemin/index.d.ts

```diff
@@ -12,7 +12,7 @@ declare namespace imagemin {
     type Plugin = (input: Buffer) => Promise<Buffer>;
 
     interface Options {
-        destination?: string;
+        destination: string;
```

Comment:
[`destination` should be optional](https://github.com/imagemin/imagemin#destination)

Reply:
Yes, you are correct. Just tested and it is optional. Sorry for the chaos.

Reply:
Sorry, you need to approve it again. I reverted last commit. @romain-faust